MUI/src/main/java/mui/MUIRunner.java

```diff
@@ -23,11 +27,33 @@ public void stopProc() {
 		isTerminated = true;
 	}
 
-	public void callProc(String[] manticoreArgs) {
+	public void constructCommand(String manticoreExePath,
```

Comment:
This doesn't seem to be used anywhere? Looks like `buildFormPanel` has some similar logic

Reply:
!! forgot to remove that